src/app-layout/drawer/drawers-helpers.ts

```diff
@@ -4,20 +4,14 @@
 export function splitItems<T extends { id: string }>(
   maybeItems: Array<T> | undefined,
   splitIndex: number,
-  activeId: string | null | undefined,
-  isMobile = false
+  activeId: string | undefined
 ) {
   const items = maybeItems ?? [];
-  let visibleItems = items.slice(0, splitIndex);
-  let overflowItems = items.slice(splitIndex, items.length);
+  const visibleItems = items.slice(0, splitIndex);
+  const overflowItems = items.slice(splitIndex);
 
   if (overflowItems.length === 1) {
-    visibleItems = items.slice(0, splitIndex + 1);
```

Comment:
Removing potentially confusing `.slice` from here. The intent of this code is to pass down all `items` as visible